benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -0,0 +1,62 @@
+import csv
+from abc import ABC, abstractmethod
+
+from benchmarks.dynamo.pr_time_benchmarks.cirron.cirron import Collector
```

Comment:
I don't see benchmarks.dynamo.pr_time_benchmarks.cirron.cirron ?

Also if you copy cirron, you should probably copy into third_party, see https://github.com/pytorch/pytorch/tree/main/third_party/miniz-2.1.0 for example.
And make sure all the licenses are in place.

Reply:
looks like i forgot to add it, question if i copy it there would it not sync? I am also modifying it alot just copying two files from it